tests/ui_tests/app/page/common/configuration/page.py

```diff
@@ -189,6 +198,33 @@ def _click_group():
 
         wait_until_step_succeeds(_click_group, period=1, timeout=10)
 
+    @contextmanager
+    def wait_group_changed(self, group_name: str):
```

Comment:
add docstring
it's strange that linter doesn't found it

Reply:
added